bundles/framework/layerlist/view/LayerViewTabs/LayerIcon.jsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import { Message, Tooltip } from 'oskari-ui';
 import PropTypes from 'prop-types';
-import { ImageLayerIcon, DataLayerIcon, UserDataIcon, ThreeDIcon } from './CustomIcons';
+import React from 'react';
+import { DataLayerIcon, ImageLayerIcon, ThreeDIcon, TimeSerieIcon, UserDataIcon } from './CustomIcons';
 
 // TODO replace logic with layer plugins registering their icons.
-export const LayerIcon = ({ type, ...rest }) => {
+export const LayerIcon = ({ type, hasTimeseries = false, ...rest }) => {
+    if (hasTimeseries) {
+        return (
+            <Tooltip title={<Message messageKey="layer.tooltip.timeseries" />}>
```

Comment:
Let's remove the tooltip from here and just use the TimeSerieIcon and pass the ...rest for it like the other icons.